### PR TITLE
Push images to DockerHub fluxcd org

### PR DESCRIPTION
**Pull request by hiddeco**
This includes the change of the default image name from `weaveworks/*` to `fluxcd/*`.

- [x] Needs a new bot account, with access to the organization, the credentials are expected to be environment variables in CircleCI (`DOCKER_FLUXCD_USER`, `DOCKER_FLUXCD_PASSWORD`).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -60,13 +60,22 @@ jobs:
...
-              docker tag "docker.io/weaveworks/flux:$(docker/image-tag)" "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
+              docker tag "docker.io/fluxcd/flux:$(docker/image-tag)" "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
               docker push "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
 
-              docker tag "docker.io/weaveworks/helm-operator:$(docker/image-tag)" "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
+              docker tag "docker.io/fluxcd/helm-operator:$(docker/image-tag)" "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
               docker push "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
+
+              # Push to fluxcd org
+              echo "$DOCKER_HUB_PASSWORD" | docker login --username "$DOCKER_HUB_USER" --password-stdin
```
Comment (squaremo) [inline .circleci/config.yml]: It's confusing having one set of creds as `DOCKER_REGISTRY_*` and one as `DOCKER_HUB_*`. Could the alternate creds be `DOCKER_FLUXCD_*` or something like that?

File: .circleci/config.yml [hunk 2]
```diff
@@ -75,12 +84,25 @@ jobs:
...
-              echo "$DOCKER_REGISTRY_PASSWORD" | docker login --username "$DOCKER_REGISTRY_USER" --password-stdin
               RELEASE_TAG=$(echo "$CIRCLE_TAG" | cut -c 6-)
+
+              # Push to weaveworks org
+              echo "$DOCKER_REGISTRY_PASSWORD" | docker login --username "$DOCKER_REGISTRY_USER" --password-stdin
+              docker tag "docker.io/fluxcd/helm-operator:${RELEASE_TAG}" "docker.io/weaveworks/helm-operator:${RELEASE_TAG}"
+              docker push "docker.io/weaveworks/helm-operator:${RELEASE_TAG}"
+
+              # Push to fluxcd org
+              echo "$DOCKER_HUB_PASSWORD" | docker login --username "$DOCKER_HUB_USER" --password-stdin
```
Comment (squaremo) [inline .circleci/config.yml]: This one appears to use the fluxcd creds, but push to weaveworks/

File: .circleci/config.yml (hunk 1 above)
Comment (hiddeco) [reply .circleci/config.yml]: Good point, changed it to `DOCKER_FLUXCD_*`.

File: .circleci/config.yml [hunk 3]
```diff
@@ -60,13 +60,22 @@ jobs:
...
-              docker tag "docker.io/weaveworks/flux:$(docker/image-tag)" "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
+              docker tag "docker.io/fluxcd/flux:$(docker/image-tag)" "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
               docker push "docker.io/weaveworks/flux-prerelease:$(docker/image-tag)"
 
-              docker tag "docker.io/weaveworks/helm-operator:$(docker/image-tag)" "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
+              docker tag "docker.io/fluxcd/helm-operator:$(docker/image-tag)" "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
               docker push "docker.io/weaveworks/helm-operator-prerelease:$(docker/image-tag)"
+
+              # Push to fluxcd org
+              echo "$DOCKER_FLUXCD_PASSWORD" | docker login --username "$DOCKER_FLUXD_USER" --password-stdin
```
Comment (squaremo) [inline .circleci/config.yml]:
```suggestion
              echo "$DOCKER_FLUXCD_PASSWORD" | docker login --username "$DOCKER_FLUXCD_USER" --password-stdin
```

**Pull request merged**